### PR TITLE
Unescape Characters When Printing

### LINKED ISSUE #5: Quotes Are Unescaped And Left As HTML Number
Example:

```➜  define git:(master) define benevolent

  benevolent

    1. altruistic or charitable.
    2. generous.
    3. altruistic, charitable, good, just and fair.
    4. Having a disposition to do good
    5. generous in providing aid to others
    6. showing or motivated by sympathy and understanding and generosity; &#34;was charitable in his opinions of others&#34;; &#34;kindly criticism&#34;; &#34;a kindly act&#34;; &#34;sympathetic words&#34;; &#34;a large-hearted mentor&#34;
    7. Possessing or manifesting love for mankind.
    8. Possessing or manifesting love for mankind
    9. intending or showing kindness; &#34;a benevolent society&#34;
    10. Having a disposition to do good.
    11. altruistic or charitable
    12. generous in assistance to the poor; &#34;a benevolent contributor&#34;; &#34;eleemosynary relief&#34;; &#34;philanthropic contributions&#34;


  ---------------------------------
  Results provided by: "Glosbe API"
```

I believe all you have to do is unescape the text during writing.

### PR DESCRIPTION
**Pull request by jshah**
Fixes #5 

screen shot 2018-08-10 at 9 45 25 pm

### CODE REVIEW

**Comment by jshah** [comment]
@Rican7 would love a review when you have some time

**Comment by jshah** [comment]
@Rican7 is there anything more I need to do to get this merged in?

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+
 # Vendor Directory
 /vendor
 
 # Build Directory
 /.tmpbuild
+
+# IntelliJ
+*.iml
+.idea/
```
Comment (Rican7) [inline .gitignore]: Personal tooling and editor-specific ignores should be in your personal "global gitignore". Adding them to a specific project isn't realistically maintainable.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,12 @@
+# Define binary
+define
```
Comment (Rican7) [inline .gitignore]: Good call. The `Makefile` dictates a name for a release build-directory, but a typical build will still spit out a binary with this name. 👍

File: internal/io/writer.go [hunk 3]
```diff
@@ -82,7 +83,7 @@ func (w *PanicWriter) WriteNewLine() int {
 // character after the given string, and returns the number of bytes that were
 // written. It'll panic if any error occurs during writing.
 func (w *PanicWriter) WriteStringLine(p string) int {
-	return w.WriteString(p) + w.WriteNewLine()
+	return w.WriteString(html.UnescapeString(p)) + w.WriteNewLine()
```
Comment (Rican7) [inline internal/io/writer.go]: Hmm. I see what this is intending, but HTML, especially format-specific structures, should _never_ be in the fetched data from the source, regardless of the output style. Even if we were to have a non-console writer (like something that maybe exported the data into an HTML page or something), the structuring would be done in a separate place.

I guess what I'm trying to say is that HTML escaping shouldn't be done at the writer level, here, but instead should be fixed at the specific source that's failing to clean the fetched data. Does that make sense?

For example, the `webster` source currently attempts to clean certain strings:
https://github.com/Rican7/define/blob/00c33da75ed272debbe7f2ed3b6bdd10043ff9c0/source/webster/webster.go#L451-L454

We should do cleaning/sanitizing of sorts in the source, not the writer.

Comment (jshah) [reply internal/io/writer.go]: @Rican7 what's the point of sanitizing and then unescaping? Don't those cancel each other out?

**Comment by jshah** [comment]
@Rican7 have updated the PR

**Pull request merged**